socorro/external/boto/crashstorage.py

```diff
@@ -293,12 +304,37 @@ def save_raw_and_processed(
         processed_crash,
         crash_id
     ):
-        schema = PROCESSED_CRASH_JSON_SCHEMA
-        crash = json_schema_reducer.make_reduced_dict(schema, processed_crash)
+        all_fields = SuperSearchFields(config=self.config).get()
```

Comment:
ah! that old problem again. 
So, does introducing 1 ES query for every single crash processed introduce a big overhead. Also, ES is notoriously flaky so I'm afraid that every time it fails this read, we either have to skip or it breaks the whole processing. 

What about an optimizing like:

```
all_fields = None
if hasattr(self, 'all_fields') and hasattr(self, 'all_fields_timestamp'):
    if time.time() - self.all_fields_timestamp > 60 * 60:
        # cached but too old
        self.all_fields = None
 if not self.all_fields:
    self.all_fields = SuperSearchFields(config=self.config).get()
    self.all_fields_timestamp = time.time()
all_fields = self.all_fields
```
What do you think?